RecoMET/METProducers/src/BeamHaloSummaryProducer.cc

```diff
@@ -258,7 +258,7 @@ void BeamHaloSummaryProducer::produce(Event& iEvent, const EventSetup& iSetup)
     {
       if( iWedge->NumberOfConstituents() > T_EcalPhiWedgeConstituents )
         GlobalTightId = true;
-      if( std::abs(iWedge->ZDirectionConfidence() > T_EcalPhiWedgeConfidence) )
+      if( std::abs(iWedge->ZDirectionConfidence()) > T_EcalPhiWedgeConfidence )
```

Comment:
@cvuosalo 
you said ``` There should be no change in monitored quantities. ```
(... also in other similar PRs, did we have edits like this?)
The edit like the one here changes the logic.
Did you check how significant was it? Mainly at least for the record of having the effects of a bugfix.

@lathomas please clarify if we should expect a significant effect here.
This part of the code is old and it's unclear if it's executed or if it's contributing to used beam halo IDs.